scripts/weblate/double_languages.py

```diff
@@ -60,7 +60,7 @@
     except Exception as err:
         print('Other error occurred while notifying Mattermost channel: '+err.message+' '+err.args)
     try:
-        responseWeblate=w.request('post','https://translate.mattermost.com/api/components/i18n-wip/'+projects[project]['wip']+'/lock/',{'lock':True})
+        responseWeblate=w.request('post','https://translate.mattermost.com/api/components/i18n-wip/'+projects[project]['wip']+'/lock/',{'lock':True
```

Comment:
I think the last two characters of this statement (`})`) were cut off. You might want to double-check.

Reply:
mmm, strange, how did that happen...
But solved it